### ISSUE TITLE
GraphQL default variable not working as expected [cors] [only-browser] 

### ISSUE THREAD

**Issue by gjeanmart** [graphprotocol/graph-node#1770]
**Do you want to request a *feature* or report a *bug*?** bug

**What is the current behavior?**

The following GraphQL query fails when `null` is passed to the variables `$blockStart` and `$skip`

- endpoint: https://api.thegraph.com/subgraphs/name/multis/multisig-ropsten
- variables: `{"id": "0x573d9fce703275d49e9b68ac168133dba2253b58", "blockStart": null, "skip": null}`
- query:
```
query wallet($id: ID!, $blockStart: Int = 0, $skip: Int = 0) {
  wallet(id: $id) {
    id
    transactions(where: {block_gte: $blockStart}, skip: $skip, orderBy: stamp, orderDirection: desc) {
      id
      hash
      block
    }
  }
}
```

The exception raised is:

```
Access to XMLHttpRequest at 'https://api.thegraph.com/subgraphs/name/multis/multisig-ropsten' from origin 'http://localhost:8000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

To reproduce, simply test the above query in a browser.

**What is the expected behavior?**

should return a response.

**Comment by lutter** [comment]
This should not cause CORS issues. Once the CORS issue is fixed, make sure that we return a sensible error message.

**Comment by leoyvens** [comment]
Closing as a duplicate of https://github.com/graphprotocol/graph-node/issues/2193
